platform/infra/lib/infra/vpc.ts

```diff
@@ -9,8 +9,42 @@ export default class VpcStack extends Stack {
     constructor(scope: Construct, id: string, props: StackProps) {
         super(scope, id, props);
         this.vpc = new Vpc(this, "appmod-vpc", {
-            vpcName: "appmod-vpc"
+            vpcName: "appmod-vpc",
+            maxAzs: 3,
+            ipAddresses: IpAddresses.cidr('192.168.0.0/16'),
+            subnetConfiguration: [
+                {
+                  cidrMask: 19,
+                  name: 'public-subnet',
+                  subnetType: SubnetType.PUBLIC,
+                },
+                {
+                  cidrMask: 19,
+                  name: 'private-egress-subnet',
+                  subnetType: SubnetType.PRIVATE_WITH_EGRESS,
+                }
+            ]
         });
-    }
 
+        const publicSubnets = this.vpc.selectSubnets({
+            subnetType: SubnetType.PUBLIC
+        }).subnets;
+
+        const privateSubnets = this.vpc.selectSubnets({
+            subnetType: SubnetType.PRIVATE_WITH_EGRESS
+        }).subnets;
+
+        let clusterLabels: string[] = ['kubernetes.io/cluster/hybrid-cluster','kubernetes.io/cluster/dev-stage-blueprint', 'kubernetes.io/cluster/prod-stage-blueprint']
+        
+        let i = 0;
+        privateSubnets.forEach((subnet) => {
+            Tags.of(subnet).add('kubernetes.io/role/internal-elb', '1')
+            Tags.of(subnet).add(clusterLabels[i],'shared');
```

Comment:
This seems odd - looks like each cluster will be assigned to only one subnet? e.g. subnet-a will get hybrid, subnet-b will get dev, and subnet-c will get prod.

Was it your intention? In general, each subnet should be tagged with all three.  In any case, I think it must be at least two for ALB controller to work based on [this](https://repost.aws/knowledge-center/eks-load-balancer-controller-subnets#:~:text=Confirm%20that%20you%20have%20at%20least%20two%20subnets%20in%20two%20different%20Availability%20Zones.%20This%20is%20a%20requirement%20for%20creating%20an%20Application%20Load%20Balancer.).

Reply:
This is fixed in commit [4723a31](https://github.com/shapirov103/appmod-blueprints/pull/15/commits/4723a310ea37a1462eddbe39288ded3a9d4e544b)